frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Link from "../Link";
+
+export interface BreadcrumbsProps {
+  links: { title: string; href: string }[];
+}
+
+const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => {
+  // taking in the array of links as defined above, create an array of Link components
+  const arrayLinks = links.map((link, index) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <Link href={link.href} title={link.title} key={index}>
+      {link.title}
+    </Link>
+  ));
+
+  // menu requires ">" after each link except the last one
+  // provided it is an array of more than one link, build another array of Links except this time add the ">" after every single one with an exception
+  // for if you are at the last link in which case do not add anything after it
+  let menuBar = [];
+  if (arrayLinks.length > 1) {
+    for (let i = 0; i < arrayLinks.length; i += 1) {
+      if (i < arrayLinks.length - 1) {
+        menuBar.push(arrayLinks[i]);
+        menuBar.push(<span data-h2-padding="b(right-left, xs)">&gt;</span>);
+      } else {
+        menuBar.push(arrayLinks[i]);
+      }
+    }
+  } else {
+    menuBar = arrayLinks;
+  }
```

Comment:
I see what you're doing here, it looks like something that might make sense as a utility function:
```ts
/**
 * Inserts a separator between each pair of adjacent items in an array.
 * Kind of like array.join, except it leaves you with an array rather than a string.
 */
export function insertBetween<T>(separator: T, arr: T[]): T[] {
  return arr.reduce<T[]>((prev, curr, i) => {
    // When i === 0, prev is []. We only want separator between our original items, not at the beginning.
    if (i > 0) {
      prev.push(separator);
    }
    prev.push(curr);
    return prev;
  }, []);
}
```
You could add that to the helpers/util.ts file, and then call it here with:
```tsx
  // menu requires ">" between each pair of links
  const separator = <span data-h2-padding="b(right-left, xs)">&gt;</span>;
  const menuBar = insertBetween(separator, arrayLinks)
```